apps/src/applab/AppLabCrosshairOverlay.jsx

```diff
@@ -1,5 +1,6 @@
 /** @file App Lab-specific Crosshair Overlay */
 
+import React from 'react';
```

Comment:
Why does this test change result in us having to start importing React in our product code?